src/actions/account.js

```diff
@@ -168,17 +228,18 @@ export const { addAccessKey, addAccessKeySeedPhrase, clearAlert } = createAction
         (accountId, contractId, publicKey, successUrl, title) => defaultCodesFor('account.login', {title})
     ],
     ADD_ACCESS_KEY_SEED_PHRASE: [
-        async (accountId, contractName, publicKey) => {
-            const [walletReturnData] = await Promise.all([
-                wallet.addAccessKey(accountId, contractName, publicKey),
-                sendJson('POST', `${ACCOUNT_HELPER_URL}/account/seedPhraseAdded`, {
+        async (accountId, contractName, publicKey, isNew, fundingContract, fundingKey) => {
+            if (isNew) {
+                await wallet.createNewAccount(accountId, fundingContract, fundingKey)
+            }
+            const res = await wallet.addAccessKey(accountId, contractName, publicKey, true)
```

Comment:
nit: what does `true` mean? Best to save to variable first to make more clear, smth like:

```
const fullAccess = true;
await wallet.addAccessKey(accountId, contractName, publicKey, fullAccess);
```

Reply:
ok